proteinbenchmark/simulation_parameters.py

```diff
@@ -11,13 +11,16 @@
 
 # Default minimization parameters
 RESTRAINT_ENERGY_CONSTANT = 1.0 * unit.kilocalorie_per_mole / unit.angstrom**2
+ENERGY_TOLERANCE = 100 * unit.kilojoules_per_mole / unit.nanometer
```

Comment:
For engine-specific options, we should include an `OPENMM` or `GMX` prefix with variable names.